ClearKeep/Scenes/Home/Views/Toast.swift

```diff
@@ -0,0 +1,96 @@
+//
+//  Toast.swift
+//  ClearKeep
+//
+//  Created by HOANDHTB on 13/07/2022.
+//
+
+import SwiftUI
+
+struct Toast: ViewModifier {
+	// these correspond to Android values f
+	// or DURATION_SHORT and DURATION_LONG
+	static let short: TimeInterval = 2
+	static let long: TimeInterval = 3.5
+	
+	let message: String
+	@Binding var isShowing: Bool
+	let config: Config
+	
+	func body(content: Content) -> some View {
+		ZStack {
+			content
+			toastView
+		}
+	}
+
+	private var toastView: some View {
+	VStack {
+	  Spacer()
+	  if isShowing {
+		Group {
+		  Text(message)
+			.multilineTextAlignment(.center)
+			.foregroundColor(config.textColor)
+			.font(config.font)
+			.padding(8)
+		}
+		.background(config.backgroundColor)
+		.cornerRadius(8)
+		.onTapGesture {
+		  isShowing = false
+		}
+		.onAppear {
+		  DispatchQueue.main.asyncAfter(deadline: .now() + config.duration) {
+			isShowing = false
+		  }
+		}
+	  }
+	}
+	.padding(.horizontal, 16)
+	.padding(.bottom, 18)
+	.animation(config.animation, value: isShowing)
+	.transition(config.transition)
+  }
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ClearKeep/ck-ios/blob/fe9f474e9dacdb67166a3b18715e917932515a0f/ClearKeep/Scenes/Home/Views/Toast.swift#L54"></span>Please use tabs instead of spaces<br />
<code>not_spaces</code> <code>Toast.swift:54</code></td>
    </tr>
  </tbody>
</table>